doc/features/open-telemetry/tracing-rfc.md

```diff
@@ -0,0 +1,415 @@
+- Feature Name: Add OpenTelemetry Traces
+- Start Date: 2023-12-27
+
```

Comment:
One topic that seems not be covered in this RFC is the [IStatement.IsTracing](https://github.com/simaoribeiro/csharp-driver/blob/bc6863efd17a01c26c46369248205c1a35a5a6e0/src/Cassandra/IStatement.cs#L53) property. 

I am highlighting this because I remember that this property could lead to certain doubts with the OpenTelemetry implementation.

Is it not a concern anymore?